datadog_checks_base/datadog_checks/base/stubs/aggregator.py

```diff
@@ -62,6 +62,15 @@ class AggregatorStub(object):
     GAUGE, RATE, COUNT, MONOTONIC_COUNT, COUNTER, HISTOGRAM, HISTORATE = list(METRIC_ENUM_MAP.values())
     AGGREGATE_TYPES = {COUNT, COUNTER}
     IGNORED_METRICS = {'datadog.agent.profile.memory.check_run_alloc'}
+    METRIC_TYPE_SUBMISSION_TO_BACKEND_MAP = {
+        'gauge': 'gauge',
+        'rate': 'gauge',
+        'count': 'count',
+        'monotonic_count': 'count',
+        'counter': 'rate',
+        'histogram': 'rate', # Checking .count only, the other are gauges
+        'historate': 'rate', # Checking .count only, the other are gauges
```

Comment:
`historate` is not used by any integration and seems to be deprecated, but `self.historate` can still be called so I added it